### ISSUE TITLE
SBT dependency for 2.12 artifact

### ISSUE THREAD

**Issue by larsjaas** [alonsodomin/cron4s#49]
I've added the dependency

    val cron4s = "com.github.alonsodomin.cron4s" %% "cron4s" % "0.3.0"

and when SBT tries to resolve it, it fails on

    [warn] 	Note: Unresolved dependencies path:
    [warn] 		com.github.alonsodomin.cron4s:cron4s-tests_2.12:0.3.0 (/Users/larsa/Code/portfolio/build.sbt#L74-75)
    [warn] 		  +- com.github.alonsodomin.cron4s:cron4s_2.12:0.3.0 (/Users/larsa/Code/portfolio/build.sbt#L74-75)

I see cron4s-testkit_2.12 on mvnrepository.com, but not cron4s-tests_2.12.

I suspect I may only need to pull down cron4s-core though, which resolves fine.

**Comment by alonsodomin** [comment]
This is, unfortunately, a sloppy SBT configuration problem in `cron4s` side. The fix for it has already been merged in #45.

I still have not published a new release but in the meantime the solution is simple, use the following in SBT when declaring the dependency:

```
val cron4s = "com.github.alonsodomin.cron4s" %% "cron4s-core" % "0.3.0"
```

The `tests` won't get published into Maven Central as there is no value in for end users but next release won't be referencing it.

Sorry for this.

**Comment by larsjaas** [comment]
Not a big problem. I'm happy there's a 2.12-artifact to use now :)
